newm-server/src/main/kotlin/io/newm/server/features/song/repo/SongRepositoryImpl.kt

```diff
@@ -441,6 +444,8 @@ internal class SongRepositoryImpl(
             usdPrice = usdPrice.toAdaString(),
             dspPriceAda = dspPriceLovelace.toAdaString(),
             dspPriceUsd = dspPriceUsd.toBigInteger().toAdaString(),
+            mintPriceAda = mintCostBase.toBigInteger().toAdaString(),
+            mintPriceUsd = mintPriceUsd.toAdaString(),
```

Comment:
Question: is this still correct? I mean asserting USD price field to ADA string value?